docs/canonicalk8s/snap/reference/upgrading.md

```diff
@@ -0,0 +1,51 @@
+# Upgrade notes
+
+## Upgrade Instructions
+
+### 1.32 -> 1.33
+
+Updating the {{product}} snap to version 1.33 is straightforward in most cases. 
```

Comment:
I think this is not needed for reference material. I would suggest going straight to the next line 

If you are not using ...